build.sh

```diff
@@ -151,7 +152,7 @@ start_mwaa() {
     cp -r ../tests/integration/data/v_$version/* ./dags/data
 
     DOCKER_COMPOSE_PROJECT_NAME=aws-mwaa-local-runner-$version
-    docker-compose -p $DOCKER_COMPOSE_PROJECT_NAME -f ./docker/docker-compose-local.yml up -d --wait
```

Comment:
I removed `--wait` as the command exited on my computer with a message saying `container aws-mwaa-local-runner-2_5-local-runner-1 is unhealthy` but it ended up starting anyway. I recommend that the developers check the container boot progress with `docker ps` instead. I also added `docker ps` to the readme.
<img width="1022" alt="Screenshot 2024-08-08 at 00 34 15" src="https://github.com/user-attachments/assets/c5123645-b95f-46d9-8575-34ef687cef2c">